source/BaselineOfKepler/BaselineOfKepler.class.st

```diff
@@ -12,86 +12,28 @@ BaselineOfKepler >> baseline: spec [
-					with: [ spec requires: #('Buoy').
-					spec postLoadDoIt: #registerSystemInterfaces ];
-				package: 'Kepler-Time' with: [ spec requires: #('Kepler-System' 'Chalten') ];
-				package: 'Kepler-Notifications' with: [ spec requires: #('Kepler-System') ].
-			
-			spec
-				package: 'Kepler-SystemBasedTests' with: [ spec requires: #('Kepler-System' 'Kepler-Time') ];
-				package: 'Kepler-System-Tests'
-					with: [ spec
-						requires: #('Kepler-SystemBasedTests');
-						postLoadDoIt: #registerTestInterface ];
-				package: 'Kepler-Time-Tests' with: [ spec requires: #('Kepler-Time') ];
-				package: 'Kepler-Notifications-Tests' with: [ spec requires: #('Kepler-Notifications' 'Kepler-SystemBasedTests') ].
-			spec
-				group: 'Deployment' with: #('Kepler-System' 'Kepler-Time' 'Kepler-Notifications');
-				group: 'Tests' with: #('Kepler-System-Tests' 'Kepler-Time-Tests' 'Kepler-Notifications-Tests');
-				group: 'SystemBasedTests' with: #('Kepler-SystemBasedTests');
-				group: 'Development' with: #('Deployment' 'Tests' 'SystemBasedTests');
-				group: 'default' with: #('Deployment').
-			spec preLoadDoIt: #preload:package: ]
-]
-
-{ #category : #actions }
-BaselineOfKepler >> initializeSysteInterfaceNamespace [
-
-	SystemInterface ifNil: [ SystemInterface := Namespace new ]
-]
+				package: 'Kepler-System' with: [ spec requires: #('Buoy') ];
+				group: 'Core' with: #('Kepler-System').
 
-{ #category : #actions }
-BaselineOfKepler >> preload: aLoader package: aPackageSpec [
-
-	Smalltalk at: #SystemInterface ifAbsentPut: nil.
-
-
-]
-
-{ #category : #actions }
-BaselineOfKepler >> registerEventNotificationSystemInterface [
-
-	self
-		registerInterfaceAs: #EventNotificationSystemInterface
-		named: 'Event Notification System Interface'
-		declaring:
-			#(#subscribe:to:toBeNotifiedSending: #revokeSubscriptionTo:of: #notifySubscribersTo:)
-]
-
-{ #category : #actions }
-BaselineOfKepler >> registerInterfaceAs: aKey named: aName declaring: aSelectorSet [
-
-	SystemInterface
-		bind: aKey
-		to: (Interface named: aName declaring: aSelectorSet)
-]
-
-{ #category : #actions }
-BaselineOfKepler >> registerSystemInterfaces [
-
-	self
-		initializeSysteInterfaceNamespace;
-		registerTimeSystemInterface;
-		registerEventNotificationSystemInterface
-]
-
-{ #category : #actions }
-BaselineOfKepler >> registerTestInterface [
+			spec
+				package: 'Kepler-Time' with: [ spec requires: #('Core' 'Chalten') ];
+				package: 'Kepler-Notifications' with: [ spec requires: #('Core') ];
+				group: 'Extended' with: #('Kepler-Time' 'Kepler-Notifications').
 
-	self
-		registerInterfaceAs: #TestInterface
-		named: 'Test Interface'
-		declaring: #(#currentDateTime)
-]
+			spec
+				package: 'Kepler-SystemBasedTests' with: [ spec requires: #('Core') ];
+				group: 'SUnit' with: #('Kepler-SystemBasedTests').
 
-{ #category : #actions }
-BaselineOfKepler >> registerTimeSystemInterface [
+			spec
+				package: 'Kepler-System-Tests' with: [ spec requires: #('Core') ];
```

Comment:
This dependency is not necessary. Kepler-System-Tests are the Unit Test of Kepler-System, and the SUnit group adds SystemBasedTest so it can be used for writing user story tests.